demos/twitter/src/main/java/com/datatorrent/demos/twitter/RSSFeedExtractor.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright (C) 2015 DataTorrent, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datatorrent.demos.twitter;
+
+import com.datatorrent.api.DefaultOutputPort;
+import com.datatorrent.api.Context.OperatorContext;
+import com.datatorrent.api.InputOperator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+/**
+ * <p>
+ * TwitterStatusUserExtractor class.
+ * </p>
+ *
+ * @since 0.3.2
+ */
+public class RSSFeedExtractor implements InputOperator
+{
+  protected String URLToConvert = "";
+  protected BufferedReader in;
+  URL rssURL;
+  public final transient DefaultOutputPort<String> output = new DefaultOutputPort<String>();
+
+  @Override
+  public void beginWindow(long windowId)
+  {
+    // TODO Auto-generated method stub
+  }
+
+  @Override
+  public void endWindow()
+  {
+    // TODO Auto-generated method stub
+
+  }
+
+  @Override
+  public void setup(OperatorContext context)
+  {
+    try {
+      DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+      DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+      InputStream is = this.getClass().getClassLoader()
+          .getResourceAsStream("RSS.xml");
+      Document doc = null;
+      try {
+        doc = dBuilder.parse(is);
+      } catch (SAXException e) {
+        throw new RuntimeException("SAX Exception");
```

Comment:
Can you also pass the SAXException object to RuntimeException.

In case of issues it will be very useful is actual exception is not masked by RuntimeException.

You can just pass "e" as second parameter to RuntimeException constructor.